server/static/bindata_assetfs.go

```diff
@@ -102,7 +102,7 @@ func staticCssCustomCss() (*asset, error) {
 		return nil, err
 	}
 
-	info := bindataFileInfo{name: "static/css/custom.css", size: 6946, mode: os.FileMode(420), modTime: time.Unix(1641777138, 0)}
+	info := bindataFileInfo{name: "static/css/custom.css", size: 6946, mode: os.FileMode(420), modTime: time.Unix(1635422826, 0)}
```

Comment:
I'm unsure if this changes are required, I generated this file via:

`make dist`